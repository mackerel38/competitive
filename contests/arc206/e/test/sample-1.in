3
4
1 2
5 6
7 8
3 4
6
6 2 10 2
7 1 3 0
2 4 4 2
2 10 6 4
8
5 0 6 10 1 8
5 5 9 9 4 7
7 3 4 9 7 6
10 9 0 4 10 5
