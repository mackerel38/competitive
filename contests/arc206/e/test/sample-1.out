36
15
21
